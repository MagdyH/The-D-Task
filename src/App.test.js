import React from 'react';
import renderer from 'react-test-renderer';
import APP from './App';
import {Provider} from "react-redux";
import configureStore from 'redux-mock-store';

const initialState = {}
const mockStore = configureStore();
const store = mockStore(initialState);


it('should render snapshot', () => {
  const component = renderer.create(<Provider store={store}><APP /> </Provider>);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
