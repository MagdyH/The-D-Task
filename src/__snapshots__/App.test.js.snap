// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
Array [
  <div>
    <header>
      <nav
        className="navbar navbar-expand-lg navbar-light header"
      >
        <img
          alt=""
          className="d-inline-block align-top header-img mx-4"
          src="The-D-GmbH.png"
        />
        <div
          className="collapse navbar-collapse"
        >
          <ul
            className="navbar-nav"
          >
            <li
              className="nav-item"
            />
            <li
              className="nav-item"
            >
              <a
                className="nav-link text-white"
                href="/FixturesResults"
                onClick={[Function]}
              >
                Fixtures Results
              </a>
            </li>
            <li
              className="nav-item"
            >
              <a
                className="nav-link text-white"
                href="/Clubs"
                onClick={[Function]}
              >
                Clubs
              </a>
            </li>
          </ul>
        </div>
        <div
          className="mt-4"
        />
      </nav>
    </header>
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="row col-md-12"
        />
      </div>
    </div>
  </div>,
  " ",
]
`;
